### ISSUE TITLE
Document the recommended local build workflow in a README or similar

### ISSUE THREAD

**Issue by peterlefanulumsdaine** [UniMath/SetHITs#18]
This is always good practice, but would be especially helpful for the UniMath maintainers to keep this synced with UniMath upstream issues — in particular, I would be happy to prepare a PR for #17 , but first I have to figure out how to build it as intended.

**Comment by m-lindgren** [comment]
Good idea, meanwhile here's how I build the satellites using that dune supports compositional builds:

I have the satellites cloned into my UniMath clone, so it looks like this:
```
UniMath
├── GrpdHITs
├── largecatmodules
├── SetHITs
├── TypeTheory
└── UniMath
```
then from the top UniMath folder I just do `dune build SetHITs`.

Similarly `dune build UniMath` will build only UniMath, and `dune build` will build everything.
